lynx/common/serializable.py

```diff
@@ -24,7 +25,7 @@ def serialize(self) -> str:
     # This method is responsible for populating instance of an object
     # with data from `JSON`. Usually, `self` contains defaultly initialized
     # instance of the class
-    def populate(self, json_string) -> None:
+    def populate(self, json_string) -> NoReturn:
```

Comment:
Didn't know there's something such as `NoReturn`. Quite cool, nonetheless b l o a t :)

Reply:
Once missing type hints from `typing` get added, there will be no bloat :)